addons/im_livechat/static/tests/qunit_suite_tests/components/chat_window_manager_tests.js

```diff
@@ -41,7 +41,7 @@ QUnit.test('closing a chat window with no message from admin side unpins it', as
         args: [mailChannelId1],
     }, { shadow: true });
     assert.strictEqual(
-        channels[0].is_pinned,
+        channels[0].channel[0][1].isServerPinned,
```

Comment:
`[0][1]`. Please put a comment so it's understandable why we do this (give an example of shape of command)